ui/src/test/java/org/pentaho/di/ui/repository/repositoryexplorer/controllers/BrowseControllerTest.java

```diff
@@ -148,7 +148,7 @@ public void folderWithSingleDotThrowsException() throws Exception {
     when( document.createElement( PROMPTBOX ) ).thenReturn( prompt );
 
     doNothing().when( controller ).confirm( anyString(), anyString() );
-    doReturn( prompt ).when( controller ).promptForName( any( UIRepositoryObject.class ) );
+    doReturn( prompt ).when( controller ).promptForName( any() );
```

Comment:
FYI you can also use nullable( UIRepositoryObject.class ) here if you want to be more specific.  Then again, you wouldn't have needed to do this in the first place if we weren't handing it a null anyway ;)

Reply:
Done!